api/src/main/proto/stellarstation/api/v1/groundstation/groundstation.proto

```diff
@@ -266,6 +267,14 @@ message GroundStationStreamRequest {
     // - `plan_monitoring_event.ground_station_event` must be sent once at each of the transitions
     //   documented in `PlanLifecycleEvent` when executing a plan.
     StreamEvent stream_event = 4;
+
+    // The status of plan execution. The client can tell the server that it cannot continue
+    // executing the plan by sending a FAILED status. If the status is sent, the server aborts
+    // the current plan and broadcast the status to all the connected clients.
+    //
+    // This field is ignored unless the client is connected to an endpoint at the edge device
+    // (StarPass).
+    PlanExecutionStatus plan_status = 5;
```

Comment:
Do you think that the existing stream_event isn't suitable for the GS implementation, or you feel it is just not a good API to begin with?

Can you not use stream_event but just ignore the parts you don't need just now?

I don't feel strongly about it but introducing divergence now might mean more work later.

Reply:
I decided to add a new field because the comment for `stream_event` says `Satellite operators will often use these events for debugging and monitoring.` The new field is not intended for direct consumption by the satellite operator but by StarPass's internal plan execution logic.

Reply:
I decided to add a new field. The main reason is to hide it from the satellite operator. `StreamEvent` is in `transport.proto`, which is shared by satellite operator APIs and ground station APIs.